src/kvstore/RocksEngine.h

```diff
@@ -146,6 +146,14 @@ class RocksEngine : public KVEngine {
     return walPath_.c_str();
   }
 
+  const void* GetSnapshot() override {
+    return db_->GetSnapshot();
+  }
+
+  void ReleaseSnapshot(const void* snapshot) override {
+    db_->ReleaseSnapshot(reinterpret_cast<const rocksdb::Snapshot*>(snapshot));
```

Comment:
`ReleaseSnapshot`  would remove from file system?

Reply:
No, It's only a memory snapshot.

Reply:
AFAIK snapshot will create some hard link about the previous SST files why don't remove them?

Reply:
> AFAIK snapshot will create some hard link about the previous SST files why don't remove them?

https://github.com/facebook/rocksdb/wiki/Snapshot

> A snapshot captures a point-in-time view of the DB at the time it's created. Snapshots do not persist across DB restarts.